packages/shared/src/components/history/ReadingHistoryItem.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { MouseEvent, ReactElement } from 'react';
+import classNames from 'classnames';
+import Link from 'next/link';
+import { HideReadHistoryProps, ReadHistory } from '../../graphql/users';
+import XIcon from '../../../icons/x.svg';
+import classed from '../../lib/classed';
+import { Button } from '../buttons/Button';
+import { LazyImage } from '../LazyImage';
+
+interface ReadingHistoryItemProps {
+  className?: string;
+  history: ReadHistory;
+  onHide?: (params: HideReadHistoryProps) => Promise<unknown>;
+}
+
+const SourceShadow = classed(
+  'div',
+  'absolute left-5 -my-1 w-8 h-8 rounded-full bg-theme-bg-primary',
+);
+
+function ReadingHistoryItem({
+  history: { timestamp, post },
+  onHide,
+  className,
+}: ReadingHistoryItemProps): ReactElement {
+  const onHideClick = (e: MouseEvent) => {
+    e.stopPropagation();
+    onHide({ postId: post.id, timestamp });
+  };
+
+  return (
+    <Link href={`${process.env.NEXT_PUBLIC_WEBAPP_URL}posts/${post.id}`}>
+      <article
+        className={classNames(
+          'flex relative flex-row items-center py-3 pr-5 pl-9 hover:bg-theme-hover hover:cursor-pointer',
+          className,
+        )}
+      >
+        <LazyImage
+          imgSrc={post.image}
+          imgAlt={post.title}
+          className="w-16 laptop:w-24 h-16 rounded-16"
+        />
+        <SourceShadow />
```

Comment:
Why don't you use a real shadow?

Reply:
Oh, no, I have to agree 😆

Reply:
Its container has a property `overflow: hidden` which makes the shadow (outside of the element's borders) not visible, so I had to resort to manually creating an element to replicate the purpose.